dd-sdk-android/src/test/kotlin/com/datadog/android/rum/assertj/RumEventDataViewAssert.kt

```diff
@@ -57,6 +57,13 @@ internal class RumEventDataViewAssert(actual: RumEventData.View) :
         return this
     }
 
+    fun hasMeasures(assert: RumEventDataViewMeasuresAssert.() -> Unit): RumEventDataViewAssert {
```

Comment:
I am wondering how this actually works, I saw you are passing several assertions in a lambda like :
```.hasMeasures {
             hasErrorCount()
             hasUserActionCount()
      }
```
The way I see the function defined it should only execute the last assertion (hasUserActionCount()) because this is the one returned by the lambda. Is Kotlin doing something magic to group all these assertions into a new local: RumEventDataViewMeasuresAssert.() -> Unit function ?

Reply:
`RumEventDataViewMeasuresAssert.()` means that within the lambda , `this` has type `RumEventDataViewMeasuresAssert`. 
So my assertions could be read as : 

```kotlin
assertThat(view).hasMeasures {
    this.hasErrorCount()
    this.hasUserActionCount()
}
```

In theory I could chain them ( `this.hasErrorCount().hasUserActionCount()`) but it wouldn't change anything since each assertion method returns the same instance of `RumEventDataViewMeasuresAssert`

Reply:
Ok , now I get it, tks ;)